pkgs/development/python-modules/labgrid/default.nix

```diff
@@ -0,0 +1,70 @@
+{ ansicolors
+, attrs
+, autobahn
+, buildPythonPackage
+, fetchFromGitHub
+, jinja2
+, lib
+, mock
+, pexpect
+, psutil
+, pyserial
+, pytest
+, pytest-mock
+, pyudev
+, pyusb
+, pyyaml
+, requests
+, setuptools_scm
+, xmodem }:
+
+buildPythonPackage rec {
+  pname = "labgrid";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "labgrid-project";
+    repo = "labgrid";
+    rev = "v${version}";
+    sha256 = "0zyf4cpazkxlbd1acmx3i1awi4kpi43g5qdvz41w2ls66h5jwf0f";
+  };
+
+  nativeBuildInputs = [ setuptools_scm ];
```

Comment:
```suggestion
  nativeBuildInputs = [ setuptools-scm ];
```

Reply:
We are trying to normalize python package names according to https://www.python.org/dev/peps/pep-0503/#normalized-names.